components/__tests__/SingleSelect-test.js

```diff
@@ -0,0 +1,38 @@
+/* @jest-environment jsdom */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Picker } from 'native-base';
+import SingleSelect from '../SingleSelect';
+
+
+it('sets values', () => {
+  const wrapper = shallow(<SingleSelect
+    items={[
+      { label: '', value: 'not-set' },
+      { label: 'Male', value: 'male' },
+      { label: 'Female', value: 'female' },
+    ]}   
+    selectedItem="male" 
+    onChange= {()=>{}}
+    />);
+
+  expect(wrapper
+    .find(Picker.Item).first()
+    .prop('value')).toEqual('not-set');
+  expect(wrapper
+    .find(Picker.Item).first()
+    .prop('label')).toEqual('');
+
+  expect(wrapper
+    .find(Picker.Item).last()
+    .prop('value')).toEqual('female');
+  expect(wrapper
+    .find(Picker.Item).last()
+    .prop('label')).toEqual('Female');
+
+  expect(wrapper
+    .find(Picker)
+    .prop('selectedValue')).toEqual('male');
+});
```

Comment:
This test tests the first and last items, but not the middle item ("male"). Please add assertions for the value and label of the "male" item.